ptn/boozebot/botcommands/PublicHoliday.py

```diff
@@ -226,8 +226,9 @@ async def admin_override_start_timestamp(self, interaction: discord.Interaction,
     @app_commands.command(name="booze_duration_remaining", description="Returns roughly how long the holiday has remaining.")
     async def remaining_time(self, interaction: discord. Interaction):
         print(f'User {interaction.user.name} wanted to know if the remaining time of the holiday.')
-        if not ph_check():
-            await interaction.response.send_message("Pirate Steve has not detected the holiday state yet, or it is already over.", ephemeral=True)
+        await interaction.response.defer()
```

Comment:
```suggestion
        await interaction.response.defer(ephemeral=True)
```

to match previous behaviour